Applications/Spire/Source/Ui/IconButton.cpp

```diff
@@ -46,25 +46,20 @@ IconButton::IconButton(QImage icon, Style style, QWidget* parent)
 }
 
 void IconButton::keyPressEvent(QKeyEvent* event) {
-  if(!event->isAutoRepeat() && (event->key() == Qt::Key_Enter ||
-      event->key() == Qt::Key_Return)) {
-    event->accept();
-    setDown(true);
-    Q_EMIT pressed();
-    return;
+  event->accept();
```

Comment:
Is event->accept() necessary here?

Reply:
It looks like it is redundant. The key events are accepted by default (Under the 'Detailed Description' section: [QKeyEvent](https://doc.qt.io/qt-5/qkeyevent.html)).

Reply:
I think so, and we can remove accept() in the keyPressEvent function and the whole IconButton::keyReleaseEvent function.

Reply:
You're right; setDown(...) won't be called on key presses so there won't be a click signal on the key release. No need to override it.